SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Base.h

```diff
@@ -183,14 +183,27 @@ class SOFA_CORE_API Base
     void setName(const std::string& n, int counter);
 
     /// Get the type name of this object (i.e. class and template types)
+    /// Since #PR 1283, the signature has changed to "final" so it is not possible
```

Comment:
It is not final right now so people can still compile their "old code" without updating to new API.